### PR TITLE
ECOM-1441: skip reverification

### PR DESCRIPTION
**Pull request by aamir-khan**
@zubair-arbi @awais786 please review this

### CODE REVIEW

File: edx_reverification_block/xblock/static/html/reverification.html [hunk 1]
```diff
@@ -32,9 +32,9 @@
             Cancel
         </span>
       </button>
-  <input type="hidden" name="checkpoint" id="related_assessment_input" value="{checkpoint}" />
-  <input type="hidden" name="user_id" id="user_id_input" value="{user_id}" />
-  <input type="hidden" name="course_id" id="course_id_input" value="{course_id}" />
+  <input type="hidden" name="checkpoint" id="related_assessment_input" value="{{checkpoint}}" />
```
Comment (zubair-arbi) [inline edx_reverification_block/xblock/static/html/reverification.html]: nitpick: add spaces around variable name inside curly brackets e.g, `{{ checkpoint }}`

File: edx_reverification_block/xblock/tests/test_reverification_block.py [hunk 2]
```diff
@@ -53,3 +58,28 @@ def test_load_student_view(self, xblock):
...
+
+        self.assertTrue("You have skipped re-verification" in xblock_fragment.body_html())
+
+    @scenario(TESTS_BASE_DIR + '/data/basic_scenario.xml')
+    def test_load_studio_view(self, xblock):
+        """
+        Reverification XBlock basic test for verifying that the user gets some
+        HTML.
+        """
+        # Test that creating Reverification XBlock for the first time user gets
```
Comment (zubair-arbi) [inline edx_reverification_block/xblock/tests/test_reverification_block.py]: Make this docstring: `Test that the user gets warning message for configuring the Reverification XBlock for the first time`

File: edx_reverification_block/xblock/tests/test_reverification_block.py [hunk 3]
```diff
@@ -53,3 +58,28 @@ def test_load_student_view(self, xblock):
         DummyReverificationService.get_status.return_value = "submitted"
         xblock_fragment = self.runtime.render(xblock, "student_view")
         self.assertTrue('submitted' in xblock_fragment.body_html())
+
+        # User skip the re-verification
```
Comment (zubair-arbi) [inline edx_reverification_block/xblock/tests/test_reverification_block.py]: nitpick: `User skips the Reverification`

**Comment by zubair-arbi** [comment]
Just minor nitpicks, otherwise :+1:

File: edx_reverification_block/xblock/reverification_block.py [hunk 4]
```diff
@@ -100,26 +75,30 @@ def student_view(self, context=None):
...
-                    'reverification_link': reverification_link,
-                    'org': org
-                }
-            )
-            fragment.add_content(html)
-            fragment.add_javascript(self._resource("static/js/skip_reverification.js"))
-            fragment.initialize_js('SkipReverifcation')
+            if verification_status:
+                # TODO: What message will be displayed to user if it is already has any status?
+                fragment.add_content(unicode(verification_status))
```
Comment (zubair-arbi) [inline edx_reverification_block/xblock/reverification_block.py]: nitpick: `# TODO: What message will be displayed to user if it is already has a status?`

**Comment by aamir-khan** [comment]
Closing in favor of #14

**Pull request closed without merging**